appserver/ejb/ejb-http-remoting/endpoint/src/main/java/fish/payara/ejb/invoke/InvokeEJBServlet.java

```diff
@@ -72,111 +77,112 @@
 public class InvokeEJBServlet extends HttpServlet {
     private static final long serialVersionUID = 1L;
 
+    private static final Logger logger = Logger.getLogger(InvokeEJBServlet.class.getName());
+
+    @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
         response.getWriter().append("Served at: ").append(request.getContextPath());
     }
 
+    @Override
     protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        JsonObject requestPayload = readJsonObject(request.getReader());
```

Comment:
Another error case would be malformed Json

Reply:
I did not address this since the caller usually is our client which we should make sure does not use malformed JSON. While being a possible problem it does not seemed so important to handle to me.